src/bindings/c/src/CMakeLists.txt

```diff
@@ -38,20 +37,23 @@ set_target_properties(${TARGET_NAME} PROPERTIES
     )
 
 export(TARGETS ${TARGET_NAME} NAMESPACE openvino::
-       APPEND FILE "${CMAKE_BINARY_DIR}/OpenVINOTargets.cmake")
+    APPEND FILE "${CMAKE_BINARY_DIR}/OpenVINOTargets.cmake")
 
 # install
-
 ie_cpack_add_component(${OV_CPACK_COMP_CORE_C} DEPENDS core)
 ie_cpack_add_component(${OV_CPACK_COMP_CORE_C_DEV} DEPENDS ${OV_CPACK_COMP_CORE_C})
 
 install(TARGETS ${TARGET_NAME} EXPORT OpenVINOTargets
-        RUNTIME DESTINATION ${OV_CPACK_RUNTIMEDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
-        ARCHIVE DESTINATION ${OV_CPACK_ARCHIVEDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
-        LIBRARY DESTINATION ${OV_CPACK_LIBRARYDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
-        NAMELINK_COMPONENT ${OV_CPACK_COMP_CORE_C_DEV}
-        INCLUDES DESTINATION ${OV_CPACK_INCLUDEDIR}/ie)
-
-install(DIRECTORY ${InferenceEngine_C_API_SOURCE_DIR}/include/
-        DESTINATION ${OV_CPACK_INCLUDEDIR}/ie
-        COMPONENT ${OV_CPACK_COMP_CORE_C_DEV})
+    RUNTIME DESTINATION ${OV_CPACK_RUNTIMEDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
+    ARCHIVE DESTINATION ${OV_CPACK_ARCHIVEDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
+    LIBRARY DESTINATION ${OV_CPACK_LIBRARYDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
+    NAMELINK_COMPONENT ${OV_CPACK_COMP_CORE_C_DEV}
+    INCLUDES DESTINATION ${OV_CPACK_INCLUDEDIR}/openvino/c)
+
+install(DIRECTORY ${OpenVINO_C_API_SOURCE_DIR}/include/c_api
```

Comment:
We need to differentiate old and new headers on the cmake targets level

Reply:
I cannot catch your point, could you give some suggestion on how to modify thin line?